### PR TITLE
Fix system topic can not auto created

### LINKED ISSUE #9265: Create the system even if the topic auto-creation disabled
**Describe the bug**

If disabled the topic auto-creation, the system topic under the namespace also not able to be created automatically, this will result in the topic level policy not able to use.

**To Reproduce**
1. Disable the topic auto-creation
2. Start the broker
3. User pulsar-admin topics set-retention or other policy operations.
4. Got 500 from the server

**Expected behavior**
The internal system topic should be created automatically even if the topic auto-creation disabled

**Additional context**
Broker: 2.7.0

### PR DESCRIPTION
**Pull request by 315157973**
Fixes #9265

### Motivation
If disabled the topic auto-creation, the system topic under the namespace also not able to be created automatically, this will result in the topic level policy not able to use.

### Modifications
Make the system topic be automatically created at any time

### Verifying this change
unit test：
testAutoCreationDisabled

### CODE REVIEW

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java [hunk 1]
```diff
@@ -2420,6 +2421,10 @@ public boolean isAllowAutoTopicCreation(final String topic) {
     }
 
     public boolean isAllowAutoTopicCreation(final TopicName topicName) {
+        //System topic can always be created automatically
+        if (EventsTopicNames.NAMESPACE_EVENTS_LOCAL_NAME.equals(topicName.getLocalName())) {
```
Comment (zymap) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java:2425]: It should be created automatically when enabling the system topic?

Comment (315157973) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java]: Sorry, I overlooked this

**Review by codelipenghui — APPROVED** [review]

**Pull request merged**